Dockerfile

```
@@ -45,5 +45,8 @@ RUN apt-get update \
 # copy project
 COPY . .
 
+RUN python -m django compilemessages \
+    && apt-get purge --assume-yes --auto-remove gettext
```

Nitpick: `gettext` is already present in an earlier layer, so I don't think this will reduce the image size. It might be helpful for debugging or if someone wants to run this command during local development in Docker, so I vote for leaving it in the image.

```suggestion
RUN python -m django compilemessages
```